docs/REST API/rest-api.md

```diff
@@ -2511,6 +2511,10 @@ Returns the information of the specified host.
         "coordinates": [40.6799, -74.0028]
       }
     },
+    "maintenance_window": {
```

Comment:
What should the contents of this field be if there are no upcoming maintenance windows? Should the whole field be `null`, or should `timezone` still be returned and `starts_at` be `null`?

Reply:
setting field to `null` if no upcoming maintenance windows. @rachaelshaw let me know if you want something different.

Reply:
update - will omit when not present